src/onnx-mlir.cpp

```diff
@@ -71,7 +74,8 @@ int main(int argc, char *argv[]) {
     LLVM_DEBUG(llvm::dbgs() << "multithreading is disabled\n");
   }
   loadDialects(context);
-
+  setupScope.stop();
+  auto inputFileScope = rootScope.nest("[onnx-mlir] Processing of Input File");
```

Comment:
`inputFileScope` => `inputFileTiming`

I think `[onnx-mlir] Importing Input Model to MLIR` is easier to understand.